packages/core/etc/core.api.md

```diff
@@ -51,11 +54,17 @@ export enum ComparisonOperator {
     // (undocumented)
     GT = 1,
     // (undocumented)
-    LT = -1
+    LT = -1,
 }
 
 // @public (undocumented)
-export function convert<TAmount>(calculator: Calculator<TAmount>): (dineroObject: Dinero<TAmount>, newCurrency: Currency<TAmount>, rates: Rates<TAmount>) => Dinero<TAmount>;

```

Comment:
I've been noticing that this `api.md` files are being reformatted somewhere in the commit process. It looks like prettier should ignore `**/*.md` files but some how they are being reformatted. Is it reforming the `ts` code blocks inside the markdown? It would be better not to have them reformatted or reformatted before the commit. Right now it seems like the precommit hook formats documents but doesn't add them to the current commit. So the commit completes but the formatting changes are left over for the next commit or an amend.

Reply:
I've been poking around it for a few hours and for now can't find an explanation. Prettier does ignore these files so my guess is that the files might have been manually saved at some point, and some other local formatter impacted the files.

Another lead is line endings that may be different between environments. This can be solved with `.gitattributes`, I'll experiment with that.